BUG:Android demo can't run

PC works fine，but on android I got
01-22 23:16:49.077  27070-27156/? E/AndroidRuntime﹕ FATAL EXCEPTION: GLThread 8623
    Process: com.mygdx.game.android, PID: 27070
    com.badlogic.gdx.utils.SerializationException: Error reading file: models/json/scene0_model.json
            at com.badlogic.gdx.utils.Json.fromJson(Json.java:705)
            at com.mygdx.game.blender.BlenderObjectMap.deserialize(BlenderObjectMap.java:50)
            at com.mygdx.game.blender.BlenderObjectMap.addFromJson(BlenderObjectMap.java:36)
            at com.mygdx.game.blender.BlenderAssetManager.loadPlaceholders(BlenderAssetManager.java:149)
            at com.mygdx.game.GameScreen.<init>(GameScreen.java:92)
            at com.mygdx.game.GdxDemo3D.create(GdxDemo3D.java:48)
            at com.badlogic.gdx.backends.android.AndroidGraphics.onSurfaceChanged(AndroidGraphics.java:279)
            at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1519)
            at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1240)
     Caused by: com.badlogic.gdx.utils.SerializationException: Error parsing file: models/json/scene0_model.json
            at com.badlogic.gdx.utils.JsonReader.parse(JsonReader.java:77)
            at com.badlogic.gdx.utils.Json.fromJson(Json.java:703)
            at com.mygdx.game.blender.BlenderObjectMap.deserialize(BlenderObjectMap.java:50)
            at com.mygdx.game.blender.BlenderObjectMap.addFromJson(BlenderObjectMap.java:36)
            at com.mygdx.game.blender.BlenderAssetManager.loadPlaceholders(BlenderAssetManager.java:149)
            at com.mygdx.game.GameScreen.<init>(GameScreen.java:92)
            at com.mygdx.game.GdxDemo3D.create(GdxDemo3D.java:48)
            at com.badlogic.gdx.backends.android.AndroidGraphics.onSurfaceChanged(AndroidGraphics.java:279)
            at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1519)
            at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1240)
     Caused by: com.badlogic.gdx.utils.GdxRuntimeException: File not found: models/json/scene0_model.json (Local)
            at com.badlogic.gdx.files.FileHandle.read(FileHandle.java:136)
            at com.badlogic.gdx.backends.android.AndroidFileHandle.read(AndroidFileHandle.java:80)
            at com.badlogic.gdx.files.FileHandle.reader(FileHandle.java:163)
            at com.badlogic.gdx.utils.JsonReader.parse(JsonReader.java:75)
            at com.badlogic.gdx.utils.Json.fromJson(Json.java:703)
            at com.mygdx.game.blender.BlenderObjectMap.deserialize(BlenderObjectMap.java:50)
            at com.mygdx.game.blender.BlenderObjectMap.addFromJson(BlenderObjectMap.java:36)
            at com.mygdx.game.blender.BlenderAssetManager.loadPlaceholders(BlenderAssetManager.java:149)
            at com.mygdx.game.GameScreen.<init>(GameScreen.java:92)
            at com.mygdx.game.GdxDemo3D.create(GdxDemo3D.java:48)
            at com.badlogic.gdx.backends.android.AndroidGraphics.onSurfaceChanged(AndroidGraphics.java:279)
            at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1519)
            at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1240)

--- a/core/src/com/mygdx/game/blender/BlenderObjectMap.java
+++ b/core/src/com/mygdx/game/blender/BlenderObjectMap.java
@@ -46,8 +46,9 @@ class BlenderObjectMap<T extends BlenderObject> extends ArrayMap<String, Array<T
 		return objects;
 	}
 
+	@SuppressWarnings("unchecked")
 	private Array<T> deserialize(String jsonPath, Class<T> type) {
-		return (jsonPath == null) ? new Array<T>() : new Json().fromJson(Array.class, type, Gdx.files.local(jsonPath));
+		return (jsonPath == null) ? new Array<T>() : new Json().fromJson(Array.class, type, Gdx.files.internal(jsonPath));
 	}
 
 	public void add(T object) {

--- a/core/src/com/mygdx/game/objects/Ragdoll.java
+++ b/core/src/com/mygdx/game/objects/Ragdoll.java
@@ -306,7 +306,7 @@ public abstract class Ragdoll extends GameCharacter {
 
 		@SuppressWarnings("unchecked")
 		Array<BlenderEmpty> empties =
-				new Json().fromJson(Array.class, BlenderEmpty.class, Gdx.files.local(ragdollJson));
+				new Json().fromJson(Array.class, BlenderEmpty.class, Gdx.files.internal(ragdollJson));
 
 		for (BlenderEmpty empty : empties) {
 			Vector3 halfExtents = new Vector3(empty.scale);

